lib/repository/movie_repo.dart

```diff
@@ -0,0 +1,31 @@
+import 'dart:convert';
+import 'package:flutter/services.dart';
+import '../constants/constants.dart';
+import '../model/movie_data.dart';
+import './interface/movie_interface.dart';
+
+class MovieRepository extends MovieInterface {
+  List<Movie> parsedMovieList = [];
+
+  static Future<List<Movie>> parseMovies() async {
```

Comment:
This can be non-static, and even private

Reply:
Done